sale_stock_prebook/models/stock_rule.py

```diff
@@ -1,11 +1,46 @@
 # Copyright 2023 Michael Tietz (MT Software) <mtietz@mt-software.de>
 # License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
-from odoo import models
+from odoo import fields, models
 
 
 class StockRule(models.Model):
     _inherit = "stock.rule"
 
+    prebook_picking_type_id = fields.Many2one(
```

Comment:
Couldn't this name be specific to sale ? `sale_prebook_picking_type_id`

Reply:
IMO no since this mechanism can be used by hand outside the sale process